.lintrunner.toml

```diff
@@ -972,3 +972,42 @@ command = [
     'python3',
     'tools/linter/adapters/lintrunner_version_linter.py'
 ]
+
+[[linter]]
+code = 'RUFF'
```

Comment:
One nit is that there should be two comments reminding future contributors to try keep the flake8 config and ruff config in sync. Otherwise, the rule will only be enabled / disabled in one which will cause issues.

Reply:
Done